src/Core/Components/DateTime/FluentDatePicker.razor

```diff
@@ -1,7 +1,7 @@
 @namespace Microsoft.FluentUI.AspNetCore.Components
 @inherits FluentCalendarBase
 
-<FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" />
+<FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" Required=@Required TValue="DateTime?" />
```

Comment:
To be coherent with other attributes, could you add double-quotes: Required="@Required"?